Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayIntegrationTests.swift

```diff
@@ -21,6 +21,18 @@ class SentrySessionReplayIntegrationTests: XCTestCase {
         }
     }
     
+    private class TestCrashWrapper: SentryCrashWrapper {
+        var traced = true
+        
+        init(traced: Bool = true) {
+            self.traced = traced
+        }
```

Comment:
This is an NSObject subclass, so I think we'd need to call `super.init()`, although I guess you don't actually want to start all the crash reporting machinery here, you just want the answer to whether the process is being debugged. Sounds like something we should refactor out of the crash wrapper as some point. 

If you really are avoiding `super.init()` for this reason, could we comment that?
```suggestion
        init(traced: Bool = true) {
            self.traced = traced
            // not calling super.init() here as we don't actually want to install crash reporter machinery
        }
```